thesis-155/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM postgres:12.1-alpine
+
+USER postgres
+
+ENV POSTGRES_PASSWORD P@$$0rd
+
+VOLUME vol-thesis-155:/var/lib/postgresql/data
+# VOLUME /vol-thesis-155:/var/lib/postgresql/data   #สามารถใช้แบบนี้ได้เช่นกัน
+ENV WORK_DIR=/home
```

โจทย์ต้องการให้สามารถใส่ `--build-arg WORK_DIR=<<PATH>>` เพื่อให้เปลี่ยน Path ได้เวลา build ควรใช้ `ARG`

```suggestion
ARG WORK_DIR=/home
```